version.json

```diff
@@ -1,5 +1,5 @@
 {
 	"major": "0",
-	"minor": "30",
-	"tag": ""
+	"minor": "31",
+	"tag": "next"
```

Comment:
is this change intentional?

Reply:
Yes, the changes in the n4js-libs can have quite a far reaching impact.

Reply:
That means, from now on master will release tagged versions only until we decide otherwise. ok.